app/src/main/res/values/setup.xml

```diff
@@ -19,4 +19,7 @@
 <resources>
     <string name="app_name">YacGuide</string>
     <string name="app_name_long">Yet Another Climbing Guide</string>
+
+    <!-- default settings -->
+    <bool name="order_tourbook_chronologically">false</bool>
```

Comment:
@chrgernoe What do you think about the best default value?

Reply:
Because I opened issue #70, false is my preferred value.